flask-homework/views.py

```diff
@@ -0,0 +1,147 @@
+from flask import abort, request, redirect
+import random
+
+
+from app import app
+
+
+
+# 1. Створити функції для обробки таких запитів:
+# - GET /users – має повертати рандомний список імен (будь-яку кількість)
+# - GET /books – має повертати рандомний список книжок (будь-яку кількість) у вигляді html списку
+
+# 7. (необов'язкове виконання) Модифікувати функції обробники /users та /books із першого завдання таким чином,
+# щоб вони повертали точну кількість значень на основі query param
+# count: /users?count=20 має повернути 20 значень. Якщо параметр не передано — кількість має бути рандомною.
+
+@app.route('/users')
```

Comment:
nit: clearer use particular method
```suggestion
@app.get('/users')
```